createbillboardplaylist.py

```diff
@@ -311,6 +311,14 @@ def create_all(self):
             "http://www.billboard.com/charts/hot-100",
         )
 
+		# Billboard Country 50
+        self.create_playlist_from_chart(
+            "country-50",
+            "Country 50",
```

Comment:
I think the name of the chart is actually "Hot Country Songs"

```suggestion
            "Hot Country Songs",
```